odxtools/cli/list.py

```diff
@@ -2,72 +2,93 @@
+    diag_layers: List[DiagLayer] = []
+
+    if variants is None:
+        variants = diag_layer_names
+
+    for name in variants:
+        if name in diag_layer_names:
+            diag_layers.append([x for x in odxdb.diag_layers if x.short_name == name][0])
+
+        else:
+            print(f"The variant '{name}' could not be found!")
+            return
+
+    if diag_layers:
+        print("\n")
+        print(f"Overview of diagnostic layers: ")
+        print_ecu_variant_metrics(diag_layers)
+
+    for dl in diag_layers:
+        print("\n")
+        print(f"[green]Diagnostic layer:[/green] '[bold white]{dl.short_name}[/bold white]'")
+        print(f" [blue]Variant Type[/blue]: {dl.variant_type.value}")
 
         assert isinstance(dl, DiagLayer)
         all_services: List[DiagComm] = sorted(dl.services, key=lambda x: x.short_name)
 
         data_object_properties = dl.diag_data_dictionary_spec.data_object_props
-        comparams = dl.comparams
-
-        print(f"{dl.variant_type} '{dl.short_name}'")
-        print(
-            f" num services: {len(all_services)}, num DOPs: {len(data_object_properties)}, num communication parameters: {len(comparams)}."
-        )
+        comparam = dl.comparams
 
         for proto in dl.protocols:
             if (can_rx_id := dl.get_can_receive_id(proto.short_name)) is not None:
-                print(f"  CAN receive ID for protocol '{proto.short_name}': 0x{can_rx_id:x}")
+                print(
+                    f"  [blue]CAN receive ID[/blue] for protocol '{proto.short_name}': 0x{can_rx_id:x}"
+                )
 
             if (can_tx_id := dl.get_can_send_id(proto.short_name)) is not None:
-                print(f"  CAN send ID for protocol '{proto.short_name}': 0x{can_tx_id:x}")
+                print(
+                    f"  [blue]CAN send ID[/blue] for protocol '{proto.short_name}': 0x{can_tx_id:x}"
+                )
 
         if dl.description:
             desc = format_desc(dl.description, ident=2)
-            print(f" Description: " + desc)
+            print(f" [blue]Description[/blue]: " + desc)
 
         if print_global_negative_responses and dl.global_negative_responses:
-            print(f"The global negative responses of '{dl.short_name}' are: ")
+            print("\n")
+            print(f"The [blue]global negative responses[/blue] of '{dl.short_name}' are: ")
             for gnr in dl.global_negative_responses:
-                print(f" {gnr}")
+                if print_objects:
+                    print(f" {gnr}")
+                else:
+                    print(f" {gnr.short_name}")
 
         if print_services and len(all_services) > 0:
             services = [s for s in all_services if service_filter(s)]
             if len(services) > 0:
-                print(f"The services of '{dl.short_name}' are: ")
+                print("\n")
+                print(f"The [blue]services[/blue] of '{dl.short_name}' are: ")
```

Comment:
Feels like the "The" prefix is added/removed at random

Reply:
As stated above, I changed the output of 
```
print(f"{dl.variant_type} '{dl.short_name}'")
print(f" num services: {len(all_services)}, num DOPs: {len(data_object_properties)}, num communication parameters: {len(comparams)}.")
```
For the rest of the function I didn't change the text but only formats.

Reply:
@andlaus maybe a clean-up for another PR?